content/en/blog/_posts/2023-12-18-kubernetes-1-29-feature-loadbalancer-ip-mode-alpha.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: blog
+title: "Kubernetes 1.29: New (alpha) Feature, Load Balancer IP Mode for Services"
+date: 2023-12-18
+slug: kubernetes-1-29-feature-loadbalancer-ip-mode-alpha
+---
+
+**Author:** [Aohan Yang](https://github.com/RyanAoh)
+
+This blog introduces a new alpha feature in Kubernetes 1.29. 
+It provides a configurable approach to define how Service implementations, 
+exemplified in this blog by kube-proxy, 
+handle traffic from pods to the Service, within the cluster.
+
+## Background
+
+As of today, the different kube-proxy implementations(including ipvs and iptables) bind the external IP 
+of the service which type is set to LoadBalancer to each node. 
+This is achieved through iptables redirecting packets directly to the service and ipvs binding the IP 
+to one interface on the node. This feature is implemented for the following reasons:
+
+1. **Traffic path optimization:** Efficiently redirecting pod traffic sent to the load balancer IP 
+directly to the backend service by bypassing the load balancer.
+  
+2. **Handling load balancer packets:** Some load balancers send packets with the destination IP set to 
+the load balancer's IP. As a result, these packets need to be routed directly to the correct backend service 
+on the node to avoid loops.
```

Comment:
```suggestion
   the load balancer's IP address. As a result, these packets need to be routed directly to the correct backend
   (which might not be local to that node), in order to avoid loops.
```